modules/eks/cluster/remote-state.tf

```
@@ -48,7 +48,7 @@ module "eks" {
     eks_managed_node_workers_role_arns = []
     fargate_profile_role_arns          = []
     fargate_profile_role_names         = []
-    eks_cluster_identity_oidc_issuer   = ""
+    eks_cluster_identity_oidc_issuer   = "https://placeholder.example.com/id/ABCDEF00000000000000000000000000"
```

```suggestion
    # eks_cluster_identity_oidc_issuer will never exist on the first deployment because the component itself creates the OIDC provider, 
    # yet the value must not be empty when passed to the aws-ebs-csi-driver and vpc-cni addons
    eks_cluster_identity_oidc_issuer   = "https://placeholder.example.com/id/ABCDEF00000000000000000000000000" 
```

Explanation in comments for the placeholder